### PR TITLE
chore: fix test coverage

### PR DESCRIPTION
**Pull request by jogold**
----

### Pull Request Checklist

* [ ] Testing
  - Unit test added (prefer not to modify an existing test, otherwise, it's probably a breaking change)
  - __CLI change?:__ coordinate update of integration tests with team
  - __cdk-init template change?:__ coordinated update of integration tests with team
* [ ] Docs
  - __jsdocs__: All public APIs documented
  - __README__: README and/or documentation topic updated
* [x] Title and Description
  - __Change type__: title prefixed with **fix**, **feat** will appear in changelog
  - __Title__: use lower-case and doesn't end with a period
  - __Breaking?__: last paragraph: "BREAKING CHANGE: <describe what changed + link for details>"
  - __Issues__: Indicate issues fixed via: "**Fixes #xxx**" or "**Closes #xxx**"
* [ ] Sensitive Modules (requires 2 PR approvers)
  - IAM Policy Document (in @aws-cdk/aws-iam)
  - EC2 Security Groups and ACLs (in @aws-cdk/aws-ec2)
  - Grant APIs (only if not based on official documentation with a reference)

----

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license.

### CODE REVIEW

File: tools/cdk-build-tools/config/nycrc [hunk 1]
```diff
@@ -9,6 +9,7 @@
         "lcov",
         "text-summary"
     ],
+    "extension": [".js", ".ts"],
```
Comment (eladb) [inline tools/cdk-build-tools/config/nycrc]: I think we just need `.js`

Comment (jogold) [reply tools/cdk-build-tools/config/nycrc]: Also not sure about specifying both extensions

Comment (eladb) [reply tools/cdk-build-tools/config/nycrc]: I think just .js

Comment (jogold) [reply tools/cdk-build-tools/config/nycrc]: With just `.js` we are back to `Unknown%`. It works with only `.ts`...

Comment (jogold) [reply tools/cdk-build-tools/config/nycrc]: But globally the coverage is lower than before

**Comment by jogold** [comment]
https://github.com/istanbuljs/nyc#source-map-support-for-pre-instrumented-codebases

https://github.com/istanbuljs/nyc/blob/master/CHANGELOG.md#breaking-changes

**Comment by jogold** [comment]
Coverage is now back to what is was on 26632fd.

**Pull request merged**